src/main/java/com/jobs/portal/jobsportal/controller/JobController.java

```diff
@@ -32,4 +33,20 @@
     }
 
 
+    @PostMapping("/post-job")
+    public ResponseEntity<BaseResponse> postJob(@Valid @RequestBody JobPostRequest request)throws Exception{
+        Integer isPost = service.postJob(request);
+        BaseResponse baseResponse = new BaseResponse();
+        if (isPost != null) {
+            baseResponse.setResponseMessage("SUCCESS");
+            baseResponse.setResponseCode("00100");
+        }else{
+            baseResponse.setResponseMessage("something went wrong! we are looking at our end, please try again in sometime");
+            baseResponse.setResponseCode("00200");
+        }
+        baseResponse.setResponse(isPost);
```

Comment:
do not set anything in response, responseMessage and responseCode are enough

Reply:
Done